yorkie/src/main/kotlin/dev/yorkie/document/crdt/CrdtTree.kt

```diff
@@ -438,14 +441,18 @@ internal class CrdtTree(
         // 04. Find the appropriate left node. If some nodes are inserted at the
         // same position concurrently, then we need to find the appropriate left
         // node. This is similar to RGA.
-        val allChildren = realParent.allChildren
-        val index = if (isLeftMost) 0 else allChildren.indexOf(leftSibling) + 1
-
         var updatedLeftSiblingNode = leftSibling
-        for (node in allChildren.drop(index)) {
-            if (node.id.createdAt <= executedAt) break
-            updatedLeftSiblingNode = node
+        executedAt?.let {
+            val allChildren = realParent.allChildren
+            val index = if (isLeftMost) 0 else allChildren.indexOf(leftSibling) + 1
+
+            for (node in allChildren.drop(index)) {
+                if (node.id.createdAt <= executedAt) break
+                updatedLeftSiblingNode = node
+            }
+            updatedLeftSiblingNode
```

Comment:
is this line needed?

Reply:
removed in [a9b50ca](https://github.com/yorkie-team/yorkie-android-sdk/pull/156/commits/a9b50ca0d73cf902b2a49bbd0618a5e84a02db39)